docs/digger/admin/using-http-proxy.asciidoc

```diff
@@ -34,11 +35,12 @@ You can run AeroGear Digger behind a HTTP proxy server by defining the following
 You can define these variables either in an inventory file, or via CLI.
 Please refer to the http://docs.ansible.com/ansible/playbooks_variables.html[Ansible Playbook Variables Docs] for more details.
 
-NOTE: There are issues with using HTTP proxies that require authentication.
-Please check the link:known_issues[Known Issues] doc for more details.
+NOTE: There are issues with using HTTP proxy that requires authentication.
+Please check the link:known_issues[Known Issues] for more details.
 It is recommended to use an HTTP proxy that doesn't require authentication.
 
 === HTTP Proxy Usage Limitations
```

Comment:
rewrite, which tools etc.

Reply:
The tools the user could add to a  Jenkinsfile to use in the buildfarm would be a list that we could not write as it depends on the use. It is enough to say that the user would need to configure any tools that they chose to use.

Reply:
@laurafitzgerald I thought there is a finite list of tools that can be used.

Reply:
is there an finite set of tools that can be used with proxied digger and should we list them, with examples how to configure them here?

Reply:
@pmacko1 In this version of the buildfarm we are not limiting what the user can do using the Jenkinsfile so at this point the list is not finite.

Reply:
"There may be tools that require specific configuration to work with proxy . You will need to update the configuration of those tools in the Jenkinsfile, using the values from the environment variables(HTTP_PROXY, HTTPS_PROXY,...)

Q: can we/do we need to elaborate more on this
Q: which environment variables
Q: what should be modified in Jenkinsfile

Reply:
ping @laurafitzgerald @matskiv

Reply:
@pmacko1 I don't think we need to elaborate on this. Explanation of your quote:
User might use some command in his Jenkinsfile that needs internet access. To make it work in environment with proxy server user would have to add some additional parameters for such commands and he would use environment variables as values for this parameter. 
E.g: `wget ... -e use_proxy=yes -e http_proxy="$HTTP_PROXY"`.
This is how I understand it, please correct me if I am wrong :)

Reply:
@pmacko1 I agree with @matskiv here in that we don't need to elaborate (at least just yet) We have covered the configuration of some tools. We won't know what more will be required until the community starts to us it. But the list of tools is an unknown right now so a general explanation that the user needs to figure out how to configure additional tools should be sufficient right now.

Reply:
resolved